docs/src/modeling_language.md

```diff
@@ -281,89 +281,64 @@ issuing any warning.
 >
 > No clear policy has been established yet about how to deal with
 > undeclared symbols in the calibration section. Avoid them.
+### Exogenous Shocks 
 
-### Shock specification
+Exogenous shock processes are specified in the section `exogenous` . Dolo accepts various exogenous processes such as normally distributed iid shocks, VAR1 processes, and Markov Chain processes. Dolo also allows for specific types of Markov Chains such as Poisson Processes, Aging Processes, and Death Processes.
 
-The way shocks are specified depends on the type of model. They are
-constructed using a the rules for mini-languages defined in section
-\[ref\].
-
-#### Distribution
-
-For Dynare and continuous-states models, one has to specifiy a
-multivariate distribution of the i.i.d. process for the vector of
-`shocks` (otherwise shocks are assumed to be constantly 0). This is done
-in the distribution section. A gaussian distrubution (only one supported
-so far), is specified by supplying the covariance matrix as a list of
-lists as in the following example.
+Here are examples of how to define different processes. Note the use of yaml tags.
+Normal Shock: It has mean zero and variance `Sigma`. 
+```{.sourceCode .yaml}
+exogenous:!Normal
+   Sigma: [[0.016^2]]
+```
+VAR1: `rho` is the persistence (only one allowed for now). `Sigma` is the covariance matrix. (Note: if a scalar `sigma` is given, it will be converted to `[[sigma]]` to indicate that it is the variance (not covariance) of the process).  
```

Comment:
I think we should be clear that a scalar `sigma` is also not the standard deviation. That field will always represent a variance for a scalar process or a covariance matrix for a multivariate process.